apps/src/templates/teacherNavigation/lessonMaterials/LessonMaterialsContainer.tsx

```diff
@@ -174,22 +205,33 @@ const LessonMaterialsContainer: React.FC<LessonMaterialsContainerProps> = ({
   const renderHeader = () => {
     return (
       <div className={styles.lessonMaterialsPageHeader}>
-        <SimpleDropdown
-          labelText={i18n.chooseLesson()}
-          isLabelVisible={false}
-          onChange={event => onDropdownChange(event.target.value)}
-          items={lessonOptions}
-          selectedValue={selectedLesson ? selectedLesson.id.toString() : ''}
-          name={'lessons-in-assigned-unit-dropdown'}
-          size="s"
-          id="ui-test-lessons-in-assigned-unit-dropdown"
-        />
+        <div className={styles.lessonMaterialsDropdowns}>
+          <UnitSelectorV2
```

Comment:
I am a bit worried about how this accounts for standalone units.

Reply:
Just confirmed that this doesn't quite work for (some) stand alone units:

<img width="902" alt="image" src="https://github.com/user-attachments/assets/6cc56f5f-bb6c-4113-bc57-b675a45d89c0" />

Reply:
It does work for AI for Oceans... 

<img width="888" alt="image" src="https://github.com/user-attachments/assets/71cda43c-7372-4b6f-95c2-5844940533b4" />

Reply:
Good catch here. The failure is when that section has no students. The only way to show this correctly for sections with no students would be to completely rewrite all the logic. I think standalone units should work just fine.

To fix the issue, I made the dropdown disappear instead of showing a forever loading screen. 
![image](https://github.com/user-attachments/assets/40884ca3-a27d-4451-ab3c-0a2fa6c73ccf)